recipes-devtools/rust/cargo-bin.inc

```diff
@@ -30,3 +30,8 @@ python () {
 }
 
 BBCLASSEXTEND += "native nativesdk"
+
+FILES_${PN} += "/usr/lib/rustlib/manifest-cargo /usr/etc/bash_completion.d /usr/share/zsh/site-functions/_cargo"
```

Comment:
Instead of including the `bash_completion.d` and `_cargo` files, they should probably be deleted during the `do_install` phase. They aren't really needed for non-interactive use.

The `manifest-cargo` file is surprising since there should be a bunch of other files installed in `/usr/lib/rustlib` - I'm surprised this hasn't errored before. We should just add the whole `/usr/lib/rustlib` to `$FILES`